### PR TITLE
Feature email login

### PR DESCRIPTION
**Pull request by mgajanssen**

### CODE REVIEW

File: src/main/java/ch/wisv/areafiftylan/Application.java [hunk 1]
```diff
@@ -45,14 +48,14 @@ public static void main(String[] args) {
...
      */
-    @Bean
-    @Profile("dev")
-    CommandLineRunner init(UserRepository accountRepository, TicketRepository ticketRepository, SeatService seatService,
-                           TeamRepository teamRepository) {
-
-        return new TestDataRunner(accountRepository, ticketRepository, teamRepository, seatService);
-
-    }
+//    @Bean
```
Comment (skamoen) [inline src/main/java/ch/wisv/areafiftylan/Application.java]: I think this was not intentional?

File: src/test/java/ch/wisv/areafiftylan/SeatRestIntegrationTest.java [hunk 2]
```diff
@@ -59,8 +59,8 @@
     @Autowired
     TicketRepository ticketRepository;
 
-    private User teamCaptain;
-    private String teamCaptainCleartextPassword = "password";
+    private User captian;
```
Comment (skamoen) [inline src/test/java/ch/wisv/areafiftylan/SeatRestIntegrationTest.java]: typo

File: src/main/java/ch/wisv/areafiftylan/Application.java (hunk 1 above)
Comment (mgajanssen) [inline src/main/java/ch/wisv/areafiftylan/Application.java]: Correct! Whoops...

File: src/main/java/ch/wisv/areafiftylan/users/controller/UserRestController.java [hunk 3]
```diff
@@ -164,4 +165,9 @@ public Boolean checkUsernameExists(@RequestParam String username) {
     public ResponseEntity<?> handleDataIntegrityViolationException(DataIntegrityViolationException ex) {
         return createResponseEntity(HttpStatus.CONFLICT, "Username or Email already taken!");
     }
+
+    @ExceptionHandler(ConstraintViolationException.class)
+    public ResponseEntity<?> handleConstraintViolationException(ConstraintViolationException ex) {
+        return createResponseEntity(HttpStatus.BAD_REQUEST, "Username should be a valid Email!");
```
Comment (skamoen) [inline src/main/java/ch/wisv/areafiftylan/users/controller/UserRestController.java]: Is this the only possible cause for this Exception? ConstraintViolationException sounds like it's more generic then just an invalid email field.

**Pull request merged**